Extension/.vscode/settings.json

```diff
@@ -41,8 +41,8 @@
         "editor.formatOnSave": true,
         "files.insertFinalNewline": true,
         "editor.codeActionsOnSave": {
-            "source.fixAll.eslint": true,
-            "source.organizeImports": true
+            "source.fixAll.eslint": "explicit",
```

Comment:
this kept getting auto-changed on my workspace, so it seems it should be checked in.

Reply:
Yes, that's VS Code's settings porting, i.e. they changed it from a bool to an enum.